assets/js/hooks/cell_editor/live_editor.js

```diff
@@ -541,6 +542,35 @@ export default class LiveEditor {
       .catch(() => null);
   }
 
+  /** @private */
+  jumpToDefinition(view) {
+    const pos = view.state.selection.main.head;
+    const line = view.state.doc.lineAt(pos);
+    const lineLength = line.to - line.from;
+    const text = line.text;
+
+    const column = pos - line.from;
+    if (column < 1 || column > lineLength) return null;
+
+    return this.connection
+      .intellisenseRequest("definition", { line: text, column })
+      .then((response) => {
+        if (response.contents.length > 0) {
+          const search = response.contents[0];
+          const params = new URLSearchParams(`?${search}`);
+          const line = parseInt(params.get("line"), 10);
+          const [_filename, cellId] = params.get("file").split("#cell:");
+
+          globalPubsub.broadcast("jump_to_editor", { line, cellId });
```

Comment:
I'm not sure if this is the best approach here, I wonder if there's a better way to communicate with the Session hook.

Reply:
Sounds good, but let's pass filename and line, so that we can share the handling with the link click that session already does :D